roadmap/implementers-guide/src/node/backing/statement-distribution.md

```diff
@@ -1,107 +1,410 @@
+    validators or from a collator.
+  - The reason for this is that sending a statement (which is always a
+    `CompactStatement` carrying nothing but a hash and signature) to the
+    cluster, is also signal that you are available to request the candidate
+    from.
+  - This makes the protocol easier to reason about, while also reducing network
+    messages about candidates that don't really exist.
+- Validators in a cluster receiving messages about unknown candidates request
+  the candidate (and statements) from other cluster members which have it.
+- Spam considerations
+  - The maximum depth of candidates allowed in asynchronous backing determines
+    the maximum amount of `Seconded` statements originating from a validator V
+    which each validator in a cluster may send to others. This bounds the number
+    of candidates.
+  - There is a small number of validators in each group, which further limits
+    the amount of candidates.
+- We accept candidates which don't fit in the fragment trees of any relay
+  parents.
+  - We listen to prospective parachains subsystem to learn of new additions to
+    the fragment trees.
+  - Use this to attempt to import the candidate later.
+
+### Grid Mode
+
+- Every consensus session provides randomness and a fixed validator set, which
+  is used to build a redundant grid topology.
+  - It's redundant in the sense that there are 2 paths from every node to every
+    other node.
+- This grid topology is used to create "sending" and "receiving" paths from each
+  validator group to every validator.
+- When a node observes a candidate as backed, it sends a
+  `BackedCandidateManifest` to their "receiving" nodes.
+- If receiving nodes don't yet know the candidate, they request it.
+- Once they know the candidate, they respond with a
+  `BackedCandidateAcknowledgement`.
+- Once two nodes perform a manifest/acknowledgement exchange, they can send
+  `Statement` messages directly to each other for any new statements they might
+  need.
+  - This limits the amount of statements we'd have to deal with w.r.t.
+    candidates that don't really exist. See "Manifest Exchange" section.
+- There are limitations on the number of candidates that can be advertised by
+  each peer, similar to those in the cluster. Validators do not request
+  candidates which exceed these limitations.
+- Validators request candidates as soon as they are advertised, but do not
+  import the statements until the candidate is part of the hypothetical
+  frontier, and do not re-advertise or acknowledge until the candidate is
+  considered both backed and part of the hypothetical frontier.
+- Note that requesting is not an implicit acknowledgement, and an explicit
+  acknowledgement must be sent upon receipt.
+
+## Statement distribution messages
+
+### Input
+
+- `ActiveLeavesUpdate`
+  - Notification of a change in the set of active leaves.
+- `StatementDistributionMessage::Share`
+  - Notification of a locally-originating statement.
```

Comment:
What does locally mean? In contrast to what?

Reply:
i.e. this statement comes from our node and should be distributed to other nodes. Thanks for calling this out, will clarified.